UnitTests/expected/Strings-File-Structured-Only.swift.out

```diff
@@ -27,6 +27,8 @@ enum L10n {
       case HeaderTitle
       /// Here you can change some user profile settings.
       case FooterText
+      /// Welcome %@! You have %d points on your account.
+      case Points(String, Int)
```

Comment:
I hope it's all there now. You're probably just waiting for the tests to pass.

I've got one question: When do you plan a new release so people can actually use this feature? I'd appreciate a quick 1.2.0 release but you may have other plans of course.